tests/foreman/api/test_repository.py

```diff
@@ -1500,6 +1500,38 @@ def test_missing_content_id(self, repo, function_sca_manifest_org, target_sat):
         )
         assert 'Candlepin job status: SUCCESS' in output, 'Failed to refresh manifest'
 
+    @pytest.mark.skipif(
+        (not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url'
+    )
+    def test_positive_sync_rpm_missing_filelists(self, target_sat):
+        """Synchronize repository with RPMs that have missing filelists with SSL verification disabled.
```

Comment:
```suggestion
        """Synchronize a yum-type repository with missing filelists.
```
Could we re-phrase a bit for better clarity? (the lists have nothing to with actual RPMs but the repo itself)